cmd/podman/system/service_abi.go

```diff
@@ -82,6 +82,12 @@ func restService(opts entities.ServiceOptions, flags *pflag.FlagSet, cfg *entiti
 		}
 	}()
 
+	// If the podman service is setup as socket activated, we don't want any containers
+	// to inherit these settings from the service.
+	for _, lEnv := range []string{"LISTEN_PID", "LISTEN_FDS", "LISTEN_FDNAMES"} {
+		os.Unsetenv(lEnv)
```

Comment:
These are also unset in the coreos/go-systemd library when the service is started. See https://github.com/containers/podman/blob/d9eb1269257490dddfae3fde2fe57c3009f94287/vendor/github.com/coreos/go-systemd/v22/activation/files_unix.go#L39 that is called during NewServer...() before Serve()

Reply:
Except I don't see activation.Files(true) ever being called.

Reply:
From `activation.Listeners()` which is called in NewServer...()

https://github.com/containers/podman/blob/d9eb1269257490dddfae3fde2fe57c3009f94287/vendor/github.com/coreos/go-systemd/v22/activation/listeners.go#L29